pkg/errors/web/middleware_test.go

```diff
@@ -32,48 +32,71 @@ func TestErrorHandling(t *testing.T) {
 
 	ttnNotFound := errors.DefineNotFound("test_not_found", "test not found")
 
-	for _, tt := range []struct {
-		Name    string
-		Handler echo.HandlerFunc
-		Assert  func(a *assertions.Assertion, res *http.Response)
+	for _, tc := range []struct {
+		Name  string
+		Error error
+		JSON  string
 	}{
 		{
-			Name:    "Echo404",
-			Handler: func(c echo.Context) error { return echo.ErrNotFound },
-			Assert: func(a *assertions.Assertion, res *http.Response) {
-				a.So(res.StatusCode, should.Equal, http.StatusNotFound)
-				b, _ := ioutil.ReadAll(res.Body)
-				a.So(string(b), should.ContainSubstring, `"namespace":"pkg/errors/web"`)
-			},
+			Name:  "Echo404",
+			Error: echo.ErrNotFound,
+			JSON: `{
+  "code": 5,
+  "details": [
+    {
+      "type_url": "type.googleapis.com/ttn.lorawan.v3.ErrorDetails",
+      "value": "Cg5wa2cvZXJyb3JzL3dlYhoJTm90IEZvdW5kIhgKFgoHbWVzc2FnZRILGglOb3QgRm91bmQ4BQ=="
```

Comment:
This isn't right. We need to render ErrorDetails as JSON, otherwise the console won't understand it.